<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPbhsForm</class>
 <widget class="QFrame" name="CPbhsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pitch, bank, heading</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Bank">
     <property name="text">
      <string>Bank (-179° to 180°):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_Bank">
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>deg.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="tb_ResetBank">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSlider" name="hs_Bank">
     <property name="minimum">
      <number>-179</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_Pitch">
     <property name="text">
      <string>Pitch (-179° to 180°):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_Pitch">
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>deg.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="tb_ResetPitch">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="hs_Pitch">
     <property name="minimum">
      <number>-179</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_Heading">
     <property name="text">
      <string>Heading (-179° to 180°):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_Heading">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="tb_ResetHeading">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/cross-white.png</normaloff>:/diagona/icons/diagona/icons/cross-white.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSlider" name="hs_Heading">
     <property name="minimum">
      <number>-179</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_GsKts">
     <property name="text">
      <string>Ground speed (kts):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sb_GsKts">
     <property name="suffix">
      <string>kts</string>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pb_SetOwnAircraft">
     <property name="text">
      <string>set</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Bank</tabstop>
  <tabstop>tb_ResetBank</tabstop>
  <tabstop>hs_Bank</tabstop>
  <tabstop>le_Pitch</tabstop>
  <tabstop>tb_ResetPitch</tabstop>
  <tabstop>hs_Pitch</tabstop>
  <tabstop>le_Heading</tabstop>
  <tabstop>tb_ResetHeading</tabstop>
  <tabstop>hs_Heading</tabstop>
  <tabstop>sb_GsKts</tabstop>
  <tabstop>pb_SetOwnAircraft</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
